ESX/Inventory HUD/README.md

```diff
@@ -59,6 +59,126 @@ TriggerServerEvent('dp_inventory:updateAmmoCount', GetHashKey(weaponModel), wepI
 local addToSlote = true
 TriggerEvent('dp_inventory:addCurrentWeapon', weaponModel, wepInfo, addToSlote)
 ```
+
+For police job you ca do like that and when they buy it its going to have a unique weapon id.
+This is clined line 1092
+```lua
+local hasWeapon = false
+for number,items in pairs(PlayerData.inventory) do
+	if items.name == v.weapon then 
+		hasWeapon = true
+	end
+end
+```
+esx_policejob/client/main.lua line 1184 replace all with the code
```

Comment:
https://github.com/esx-framework/esx_policejob/blob/9ceb66a591bca41ab10786bc795ebf04e52e2877/client/main.lua#L1184

Can you please clarify this more?

Reply:
yeah I guess you are right this is the server
![image](https://user-images.githubusercontent.com/64702468/102803552-13827d00-43c1-11eb-9172-fa0544d2fe21.png)
this is the function that you need to change with mine in the read me
![image](https://user-images.githubusercontent.com/64702468/102803611-28f7a700-43c1-11eb-86a2-1a7e26d33f2e.png)
and this is when the menu is opened you so it does not check if you have the weapon but the item 
![image](https://user-images.githubusercontent.com/64702468/102803684-4f1d4700-43c1-11eb-8910-6fbee33a57d5.png)

Reply:
hey done all what you said and this is what i receive, something wrong at
local hasWeapon = false
for number,items in pairs(PlayerData.inventory) do
	if items.name == v.weapon then 
		hasWeapon = true
	end
end
![333](https://user-images.githubusercontent.com/49865104/103676992-598c3300-4fb4-11eb-8252-e356c8d122ad.png)

Reply:
Okay then do this replace the PlayerData.inventory in the for loop with ESX.GetPlayerData().inventory so every time you open the menu it makes sure that the inventory is updated